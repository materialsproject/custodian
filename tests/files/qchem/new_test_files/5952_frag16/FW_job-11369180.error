/global/u2/s/sblau/pymatgen/pymatgen/io/cif.py:44: UserWarning: Please install optional dependency pybtex if youwant to extract references from CIF files.
  warnings.warn("Please install optional dependency pybtex if you"
DEBUG:launchpad:FW with id: 5952 is unique!
DEBUG:launchpad:Created/updated Launch with launch_id: 7667
DEBUG:launchpad:RUNNING FW with id: 5952
INFO:rocket.launcher:RUNNING fw_id: 5952 in directory: /scratch2/scratchdirs/sblau/ion_placement/block_2018-11-13-02-29-44-974177/launcher_2018-11-14-02-47-37-450838
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
INFO:rocket.launcher:Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
INFO:custodian.custodian:Run started at 2018-11-13 18:51:27.135410 in /scratch2/scratchdirs/sblau/ion_placement/block_2018-11-13-02-29-44-974177/launcher_2018-11-14-02-47-37-450838.
INFO:custodian.custodian:Custodian running on Python version 3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 18:10:19)  [GCC 7.2.0]
INFO:custodian.custodian:Hostname: nid04092, Cluster: unknown
INFO:custodian.custodian:Starting job no. 1 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 2 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 3 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 4 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 5 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 6 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Logging to custodian.json...
INFO:custodian.custodian:Run ended at 2018-11-13 19:51:58.862730.
INFO:custodian.custodian:Run completed. Total time taken = 1:00:31.727320.
