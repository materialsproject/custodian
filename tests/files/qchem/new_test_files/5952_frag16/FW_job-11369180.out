foobar
2018-11-13 18:51:23,538 INFO Hostname/IP lookup (this will take a few seconds)
2018-11-13 18:51:23,541 INFO Launching Rocket
2018-11-13 18:51:27,095 INFO RUNNING fw_id: 5952 in directory: /scratch2/scratchdirs/sblau/ion_placement/block_2018-11-13-02-29-44-974177/launcher_2018-11-14-02-47-37-450838
2018-11-13 18:51:27,126 INFO Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
2018-11-13 18:51:27,132 INFO Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
2018-11-13 18:51:27,135 INFO Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
